### DISCUSSION TITLE
Recover accidently deleted ODBC Driver

### DISCUSSION THREAD

**Discussion by Cyberbob64** [dbeaver/dbeaver#14203] [Q&A] [answered]
I accidently deleted the odbc driver. Even when i reinstall dbeaver it will not be recovered. It is not shown in the driverlist anymore. How can i recover the ODBC driver?

Thanks in advance

**Comment by Matvey16** [comment]
@Cyberbob64 Does it appear in driver manager window?
![Dzr6u9hEfW](https://user-images.githubusercontent.com/82543000/136923835-36dc681b-2b16-4ca8-8e52-8cc9f25f8580.png)
![dbeaver_bsdeOc2bJL](https://user-images.githubusercontent.com/82543000/136923844-d66fe1dc-5f3a-4268-a97d-6f715e31f179.png)

**Comment by Cyberbob64** [comment]
No, it does not appear in the driver manager :(
![grafik](https://user-images.githubusercontent.com/92361539/136926104-bb945a27-5e69-4e1b-8277-9b1199c83e7b.png)
![grafik](https://user-images.githubusercontent.com/92361539/136926202-2b846f03-d627-4245-b193-54369decc018.png)

**Comment by Matvey16** [comment, marked as answer]
@Cyberbob64 Have you tried `un-delete` function?

**Comment by Cyberbob64** [comment]
OMG ... I wear glasses but did not recognize that function :D
Now it works 
thank you
